src/setup/platformSetup/index.website.js

```diff
@@ -3,6 +3,10 @@ import {AppRegistry} from 'react-native';
 // This is a polyfill for InternetExplorer to support the modern KeyboardEvent.key and KeyboardEvent.code instead of KeyboardEvent.keyCode
 import 'shim-keyboard-event-key';
 
+// load all chunk file map when online
+import 'react-map-gl';
```

Comment:
@ntdiary do you have any suggestion here where we need load. when I am checking https://github.com/Expensify/App/blob/d89ca7043b9bc923c0599836a3571bb15d40ed7c/src/components/MapView/MapView.web.tsx#L8C36-L8C48 loading this but when we open distance page at the time only components mounted so it's loading all necessary chunk file when offline this is ignored.

Note: This will be needed only for the web and desktop. so I added on the platform setup this will call initially and load all chunk files. like shim-keyboard-event-key

Reply:
@ntdiary could you check this

Reply:
@ntdiary  MapView.web.tsx already we have the import statement. the actual problem is MapView.web.tsx will trigger only when you open a distance request page like a token api trigger. before open up the distance request page your offline means map file(chunk) is not loading.

Reply:
@ntdiary this is working  `mapLib={import('mapbox-gl')}` let me update PR